sridha2412/assignment2/q1.py

```diff
@@ -8,24 +8,26 @@ def __init__(self, data: int):
         self.left = None
         self.right = None
 
-def find_ancestors(root: Node, key: int) -> bool:
+def find_ancestors(root: Node, key: int, ancestors: list) -> bool:
     """ if key is present, return list of ancestors else return false """
     # base cases
     if root is None:
         return False
     if root.data == key:
         return True
 
+    # store all nodes in list
```

Comment:
This comment is not useful: here you rewrite the code in plain english. What is usually more interesting as a comment is not the "what" but the "why".

If the why is obvious, no need for a comment.